src/OrangePaymentSlipData.php

```diff
@@ -176,7 +207,8 @@ public function getBankingCustomerId()
         if (!$this->getWithBankingCustomerId()) {
             throw new DisabledDataException('banking customer ID');
         }
-        return $this->bankingCustomerId;
+        
+        return str_pad($this->bankingCustomerId, $this->bankingCustomerIdLength, '0', STR_PAD_LEFT);
```

Comment:
Sure, I get that. Just thought about the ``fillZeroes`` parameter. But aparently is the idea that it only applies when the banking customer ID nis not used. https://github.com/ravage84/SwissPaymentSlip/blob/0.12.1/src/OrangePaymentSlipData.php#L210

So 👍